gax-java/gax/src/main/java/com/google/api/gax/rpc/Watchdog.java

```diff
@@ -71,19 +75,32 @@ public final class Watchdog implements Runnable, BackgroundResource {
   private final ConcurrentHashMap<WatchdogStream, Object> openStreams = new ConcurrentHashMap<>();
 
   private final ApiClock clock;
-  private final Duration scheduleInterval;
+  private final java.time.Duration scheduleInterval;
   private final ScheduledExecutorService executor;
   private ScheduledFuture<?> future;
 
-  /** returns a Watchdog which is scheduled at the provided interval. */
+  /**
+   * This method is obsolete. Use {@link #createDuration(ApiClock, java.time.Duration,
+   * ScheduledExecutorService)} instead.
+   */
+  @ObsoleteApi("Use create(ApiClock, java.time.Duration, ScheduledExecutorService) instead")
```

Comment:
```suggestion
  @ObsoleteApi("Use createDuration(ApiClock, java.time.Duration, ScheduledExecutorService) instead")
```

Reply:
Also side note: It's not clicking in my head. Why can't this be
```
public static Watchdog create(
      ApiClock clock,
      java.time.Duration scheduleInterval,
      ScheduledExecutorService executor) {
    return createDuration(clock, toJavaTimeDuration(scheduleInterval), executor);
  }
``` 
instead of createDuration

Reply:
Bump

Reply:
@diegomarquezp Bumping this comment